src/platform/packages/shared/controls/controls-constants/src/controls_group_constants.ts

```diff
@@ -10,14 +10,18 @@
 /** Controls group state */
 export const CONTROLS_GROUP_TYPE = 'control_group';
 
-export const CONTROLS_LABEL_POSITION_ONE_LINE = 'oneLine';
-export const CONTROLS_LABEL_POSITION_TWO_LINE = 'twoLine';
-
-export const DEFAULT_CONTROLS_LABEL_POSITION = CONTROLS_LABEL_POSITION_ONE_LINE;
 export const DEFAULT_IGNORE_PARENT_SETTINGS = {
   ignoreFilters: false,
   ignoreQuery: false,
   ignoreTimerange: false,
   ignoreValidations: false,
 } as const;
 export const DEFAULT_AUTO_APPLY_SELECTIONS = true;
+
+/** This state is only relevant for sticky controls */
```

Comment:
I moved these to this file because they are **only** relevant when controls are in a group (i.e. sticky controls) and not when they are panels.